pkg/config/config_test.go

```diff
@@ -387,6 +387,10 @@ image_copy_tmp_dir="storage"`
 			gomega.Expect(config.Engine.NumLocks).To(gomega.BeEquivalentTo(2048))
 			gomega.Expect(config.Engine.OCIRuntimes["runc"]).To(gomega.Equal(OCIRuntimeMap["runc"]))
 			gomega.Expect(config.Containers.CgroupConf).To(gomega.BeNil())
+
+			caps, _ := config.Capabilities("", nil, nil)
+			gomega.Expect(strings.Join(caps, ",")).Should(gomega.ContainSubstring("CAP_SYS_CHROOT"))
```

Comment:
Should this also be checking for each of the rest of the items in the expected default set?

Reply:
That's a good idea. To be extra sure we're not regressing in the future.

Reply:
We do check that above, but that is using the internal list from the code. Do you want to hard code the list again in the test to make sure users change it in both places?